.github/scripts/install-icefall.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
https://github.com/k2-fsa/sherpa/blob/3f274af12c572776bbba4652a8d03cc66a70aa15/sherpa/python/sherpa/decode.py#L21

Shall we copy the implementation of `Nbest` from icefall
and don't make `icefall` a dependency of `sherpa`?

Reply:
Actually it needs also lhotse, etc. So as you prefer. I like having icefall dependencies because then once a new features is there, we can have it. At the same time, everytime something new popup in icefall, one needs to check that it does not break sherpa

Reply:
> Actually it needs also lhotse, etc. So as you prefer. I like having icefall dependencies because then once a new features is there, we can have it. At the same time, everytime something new popup in icefall, one needs to check that it does not break sherpa

Do you mean that `decode_manifest.py` depends on `lhotse`? That file is just a demo and if a user wants to run it, he/she has to install lhotse by himself/herself.

If we put `icefall` in `sherpa/__init__.py`, then the user has to install `icefall` before they can use `sherpa`.
I am worried that it may complicate the steps for installing `sherpa`.

Reply:
We have already copied `fast_beam_search_xxx` from icefall to sherpa, so I think we don't need to depend on icefall anymore.

Reply:
@I cannot refind the job but one failed because lhotse was not installed. (now that we need icefall). Let me remove the dependencies and copy the necessary files to sherpa then.

Reply:
> @i cannot refind the job but one failed because lhotse was not installed. (now that we need icefall). Let me remove the dependencies and copy the necessary files to sherpa then.

Thanks. No tests are using lhotse, I think.

Reply:
OK, so I undid the CI changes and add the files from icefall. I restarted the CI, so let see